ambari-server/src/main/java/org/apache/ambari/server/mpack/MpackManager.java

```diff
@@ -140,6 +152,7 @@ private void parseMpackDirectories() {
         }
       }
     } catch (NullPointerException|IOException e) {
+      LOG.error("Unable to parse existing mpack structure", e);
       e.printStackTrace();
```

Comment:
I assume that this was left in for a reason. We typically do not print the stack trace out to the console.

Reply:
I added this since originally I couldn't figure out what was going wrong. Since this stops ambari-server, then it's good to have it in both places.

Reply:
Oh, I see - why did I leave the PTS? I dunno - I guess we don't need it :)